test/units/TEST-87-EXTRA-FILE-DESCRIPTOR.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+at_exit() {
+    set +e
+
+    exec {TEST_FD1}<&-
+    exec {TEST_FD2}<&-
```

Comment:
Do we really need to close these? Files can be unlinked even if there's still open file descriptors to them.

Reply:
Probably not, just being cautious